src/main/java/uk/gov/digital/ho/hocs/casework/client/infoclient/InfoClient.java

```diff
@@ -94,6 +94,13 @@ public GetStandardLineResponse getStandardLine(UUID topicUUID) {
         return teams;
     }
 
+    @Cacheable(value = "InfoClientGetTeamForStageType", unless = "#result == null", key = "#stageType")
+    public UUID getTeamForStageType(String stageType) {
+        TeamDto response = restHelper.get(serviceBaseURL, String.format("/stageType/%s/team", stageType), TeamDto.class);
+        log.info("Got Team teamUUID {} for Stage {}, event: {}", response.getUuid(), stageType, value(EVENT, INFO_CLIENT_GET_TEAM_FOR_STAGE_SUCCESS));
```

Comment:
Can this event be taken out of the String.format
This is also a risk of an NPE when we do response.getUUID()

Reply:
this has been changed to pass `TeamDto` up and the check is made in the calling service.

Reply:
`response.getUuid()` The log will still throw an NPE if the response is null - not sure how realistic that is.

Reply:
I've just made a change to bring the exception throw into the client to protect the NPE